toolkit/tools/imagegen/configuration/systemconfig.go

```diff
@@ -138,6 +140,19 @@ func (s *SystemConfig) IsValid() (err error) {
 		return fmt.Errorf("invalid [KernelCommandLine]: %w", err)
 	}
 
+	// Validate that PackageRepos do not contain duplicate package repo name
+	repoNames := make(map[string]bool)
+	for _, packageRepo := range s.PackageRepos {
+		if err = packageRepo.IsValid(); err != nil {
+			return fmt.Errorf("invalid [PackageRepo]: %w", err)
+		}
+
+		if _, ok := repoNames[packageRepo.Name]; ok {
+			return fmt.Errorf("invalid [PackageRepos]: duplicate package repo names")
```

Comment:
Let's also mention the duplicate's name to make things easier for the user.

Reply:
Fixed